index.html

```diff
@@ -143,30 +150,31 @@ <h3>See Project</h3>
                 </li>
             </ul>
         </div>
-        <div class="projects">
-            <img class="img" src="./images/Snapshoot Portfolio (3).png">
+        <div class="projects reverse">
+            <img class="img" src="./images/Snapshoot Portfolio 4.png">
 
             </img>
             <ul class="projectInfo">
                 <li class="mps">
-                    <h3>Multi-Post <br> Stories</h3>
+                    <h3>Multi-Post Stories</h3>
                 </li>
                 <li class="techstack">
                     <div>
                         <ul>
-                            <li class="canopy">CANOPY</li>
-                            <li class="bed">Back End Dev</li>
+                            <li class="canopy">FACEBOOK</li>
+                            <li class="bed">Full Stack Dev</li>
                             <li class="year">2015</li>
                         </ul>
                     </div>
                 </li>
                 <li class="aboutProject">
-                    A daily selection of privately <br> personalized reads: no accounts or <br> sign-ups required.
-                </li>
+                    Experimental content creation feature that allows users to add to an existing story over the course
+                    of a day without spamming their friends. </li>
                 <li class="lang">
```

Comment:
- [ ] Please add a margin-left  `li here to achieve the correct spacing. Your spacing is too small

- [OPTIONAL] Note that this `li` tag  should be a `p` tag in order to retain its semantic meaning
_**figma**_
![space-1](https://user-images.githubusercontent.com/99656687/213411167-ecc9448f-d106-4089-a1dd-6c0d0c894401.png)

_**your design**_
![space-2](https://user-images.githubusercontent.com/99656687/213411224-17aac8e3-a5a5-4190-abf7-011f7bf0cfff.png)